src/BenchmarkDotNet/Portability/Cpu/CpuInfoFormatter.cs

```diff
@@ -33,7 +33,8 @@ private static string Format(string processorName, int? physicalProcessorCount,
                 parts.Add("1 physical core");
             if (physicalCoreCount > 1)
                 parts.Add($"{physicalCoreCount} physical cores");
-
+            if (processorFreq > 0)
+                parts.Add($", processor frequency = {processorFreq} MHz");
```

Comment:
ProcessorBrandString usually contains the information about processor frequency. We want to keep this section as small as possible, so there is no need to duplicate this info.

Reply:
If we have a difference between frequency in the brand string and the actual max frequency, probably it makes sense to patch the branch string and get something like `Intel(R) Core(TM) i5-4308U CPU @ 2.80GHz (Max: 3.30GHz)`.
What do you think?

Reply:
Hmm... It seems that `wmic CurrentClockSpeed` and `sysctl hw.cpufrequency` equal to the frequency value in the brand string. Let's call it "nominal frequency".
I don't know how to get the nominal frequency on Linux. `lscpu` is a good way to get the minimum and maximum frequency. Maybe it makes sense to introduce separate properties for such values.

Reply:
I think that the way to patch the brand string is awesome idea.

On my machine brand string doesn't match CurrentClockSpeed value
On ```wmic cpu get Name, CurrentClockSpeed /Format:List``` I have:
```
CurrentClockSpeed=3701
Name=Intel(R) Core(TM) i5-2500 CPU @ 3.30GHz
```
Of Linux - what if we measure ```lscpu```  with example from https://gist.github.com/Rizzen/1255ceb685702f29a25c11388ba96b0b and take max value? 
Anyway I discovered that max value measured under load is close to Max value from ```lscpu```  so I think we can consider max lscpu value as Frequency.

Reply:
On the `CpuInfo` level, it makes sense to introduce 3 properties: `NominalFrequency`, `MaxFrequency`, and `MinFrequency`. Next, we can provide `EffectiveFrequency` API which returns a proper value for our analyzers.
By the way, on macOS, we can get all 3 values:
```
$ sysctl -a | grep cpufreq
hw.cpufrequency: 2500000000
hw.cpufrequency_min: 2500000000
hw.cpufrequency_max: 2500000000
```